src/storage/v2/disk/storage.cpp

```diff
@@ -868,7 +869,40 @@ DiskStorage::DiskAccessor::DetachDeleteVertex(VertexAccessor *vertex) {
       std::move(deleted_edges));
 }
 
-void DiskStorage::DiskAccessor::PrefetchEdges(const auto &prefetch_edge_filter) {
+bool DiskStorage::DiskAccessor::PrefetchEdgeFilter(const std::string_view &diskEdgeKey,
+                                                   const VertexAccessor &vertex_acc, bool isOutEdge) {
```

Comment:
In general when functions receive boolean as an argument, it already indicates that it does more than one thing. That's why I preferred before explicit lambda

Reply:
Discussed on a call. Decided to have enum instead of bool